static/js/theme-map/VerticalFullPageMapOrchestrator.js

```diff
@@ -193,9 +203,43 @@ class VerticalFullPageMapOrchestrator extends ANSWERS.Component {
     searchThisAreaButtonEl.addEventListener('click', (e) => {
       this.searchThisArea();
     });
-
     this.setupMobileBreakpointListener();
     this.addMapComponent();
+    this.setFixedHeightsOnAndroid()
+  }
+
+  /**
+   * On Android browsers, opening up the keyboard will shift the contents of the entire page up,
+   * moving the map center, and thereby causing a searchOnMapMove to be triggered.
+   * The search response would then cause the page to update,
+   * and close the keyboard, making it impossible to actually type anything into the searchbar.
+   * 
+   * Setting a fixed height on elements like .Answers-mapWrapper prevents the keyboard from shifting the content
```

Comment:
is this only a problem for the Theme's map?

Reply:
yes searchOnMapMove is only a thing in the full page map